server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
What I partly mean is, this is a point in favour of continuing to run TPS or a TPS-like thing against production.  Perhaps we need to replace it with something significantly more light-weight, that just runs e.g. once an hour and checks whether it can successfully connect to sync, without doing all the many variations of tests that TPS currently runs.